===================================================================
RCS file: /cvs/glibc/libc/elf/ldd.bash.in,v
retrieving revision 1.31
retrieving revision 1.32
diff -u -r1.31 -r1.32
--- libc/elf/ldd.bash.in	2004/09/18 06:46:38	1.31
+++ libc/elf/ldd.bash.in	2004/09/21 15:38:48	1.32
@@ -75,6 +75,7 @@
   -u | --u | --un | --unu | --unus | --unuse | --unused)
     unused=yes
     shift
+    ;;
   --v | --ve | --ver)
     echo >&2 $"ldd: option \`$1' is ambiguous"
     exit 1
@@ -102,7 +103,7 @@
 add_env="LD_TRACE_LOADED_OBJECTS=1 LD_WARN=$warn LD_BIND_NOW=$bind_now"
 add_env="$add_env LD_VERBOSE=$verbose"
 if test "$unused" = yes; then
-  add_env="$add_env LD_DEBUG="$LD_DEBUG${LD_DEBUG:+,}unused"
+  add_env="$add_env LD_DEBUG=\"$LD_DEBUG${LD_DEBUG:+,}unused\""
 fi
 case $# in
 0)
